services/src/main/java/org/exoplatform/news/storage/jcr/JcrNewsStorage.java

```diff
@@ -1267,5 +1271,18 @@ private Session getSession(SessionProvider sessionProvider) throws RepositoryExc
                                                        .getDefaultWorkspaceName(),
                                       repositoryService.getCurrentRepository());
   }
-  
+
+  private  void updateNewsImagesPermissions(News news ,  SessionProvider sessionProvider, Space space  ) throws RepositoryException {
+    Matcher matcher = IMAGE_SRC_PATTERN.matcher(news.getBody());
+    while (matcher.find()) {
+      String match = matcher.group(1);
+      String imageUUID = match.substring(match.lastIndexOf("/") + 1);
+      ExtendedNode image = (ExtendedNode) getNodeById(imageUUID, sessionProvider);
```

Comment:
Could you do a check here if the content is really present or not ?

Reply:
done